### DISCUSSION TITLE
蓝灯续费后持续无法使用发邮件也得不到答复

### DISCUSSION THREAD

**Discussion by chinaJeffrey1** [getlantern/lantern-forum-cn#1054] [Q&A]
447284217@qq.com 我的这个账号8月续费1年的费用，但是到现在都无法使用请帮忙退款或者让我的账号能够使用。
![1111](https://github.com/user-attachments/assets/81ff63b7-cf71-4972-8f87-0411a78292fd)

**Comment by MaxMKMax** [comment]
@chinaJeffrey1 您好. 你要反馈的话最好是来论坛找我. 邮箱那里的团队忙不过来, 很少回复的.
我会帮您提交问题上去. 请问您用不了是具体什么问题? 是登入不进去专业版还是什么问题呢?

**Comment by chinaJeffrey1** [comment]
你好！购买完毕后，就是登陆不了，输入邮箱提示没有购买专业版，请协助查询一下是什么原因！谢谢






李杰夫
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月3日(星期二) 凌晨0:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 蓝灯续费后持续无法使用发邮件也得不到答复 (Discussion #1054)





 
@chinaJeffrey1 您好. 你要反馈的话最好是来论坛找我. 邮箱那里的团队忙不过来, 很少回复的.
 我会帮您提交问题上去. 请问您用不了是具体什么问题? 是登入不进去专业版还是什么问题呢?

**Comment by MaxMKMax** [comment]
@chinaJeffrey1 https://lantern.io/zh-Hans/faq
您好. 请您查看这个链接怎么登入专业版. 如果登入不了, 就是您没收到激活码.
那我会在帮您提交问题上去处理. 谢谢您的耐心等待和支持蓝等.
